package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hackhate2020-be-mern",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node app.js"
+  },
+  "dependencies": {
+    "cookie-parser": "~1.4.4",
+    "cors": "^2.8.5",
+    "debug": "~2.6.9",
+    "dotenv": "^9.0.0",
+    "express": "~4.16.1",
+    "mongoose": "^5.12.7",
+    "morgan": "~1.9.1"
+  },
+  "devDependencies": {
+    "eslint": "^7.25.0"
```

Comment:
Can you check if there are any other standard eslint packages for express and add those in i.e. airbnb  base or similar

Reply:
Airbnb base in particular